### PR TITLE
Cell types and other tests

### PR DESCRIPTION
**Pull request by cszsol**

### CODE REVIEW

File: swarm_copy_tests/test_resolving.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        },
+        {
+            "label": "Spinal Cord Ventral Horn Motor Neuron Gamma",
+            "id": "http://uri.interlex.org/base/ilx_0110943",
+        },
+        {
+            "label": "Hypoglossal Nucleus Motor Neuron",
+            "id": "http://uri.interlex.org/base/ilx_0105169",
+        },
+    ]
```
Comment (WonderPG) [inline swarm_copy_tests/test_resolving.py:175]: Three tests are missing from the original test file:
- test_resolve_query
- test_escape_punctuation
- test_failing_escape_punctuation

File: swarm_copy_tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    )
+    assert RegionMeta_test.name_[3] == "Primary Motor Cortex"
+
+    # check parents / childrens.
+    assert RegionMeta_test.parent_id[1] == 2
+    assert RegionMeta_test.parent_id[2] == 0
+    assert RegionMeta_test.parent_id[3] == 2
+    assert RegionMeta_test.children_ids[1] == []
+    assert RegionMeta_test.children_ids[2] == [1, 3]
+    assert RegionMeta_test.children_ids[3] == []
```
Comment (WonderPG) [inline swarm_copy_tests/test_utils.py:189]: Couple of tests are missing from the original file:
- test_RegionMeta_load_real_file
- test_get_file_from_KG_errors
- test_get_file_from_KG
- test_get_kg_data_errors
- test_get_kg_data
- test_is_lnmc

File: swarm_copy_tests/test_resolving.py (hunk 1 above)
Comment (kanesoban) [reply swarm_copy_tests/test_resolving.py:175]: Added them.

**Review by BoBer78 — APPROVED** [review]
Nothing to say, looks good !

**Pull request merged**